src/linear_algebra/matrix/nonsingular_inverse.lean

```diff
@@ -624,3 +625,30 @@ by rw [det_one_add_mul_comm, det_unique, pi.add_apply, pi.add_apply, matrix.one_
 end det
 
 end matrix
+
+namespace matrix
```

Comment:
Can you merge this with the namespace above? You should be able to remove all the duplicate variable lines too.